elm.json

```diff
@@ -1,49 +1,31 @@
 {
-    "type": "application",
-    "source-directories": [
-        "src/compiler",
-        "src/library",
-        "src/cli"
+    "type": "package",
```

Comment:
Change from application to package, because for libraries the filepaths are kinda set in stone... top-level elm.json, src/ source directory...